manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -41,6 +41,10 @@ spec:
         prometheus.io/scrape: 'true'
         prometheus.io/port: "15014"
         prometheus.io/path: '/metrics'
+        # Add AppArmor annotation
```

Comment:
Can we please be explicit about the K8S versions this is relevant/required for here, at least, and mention that it's directly analogous to the securityContext setting, which would be otherwise preferred.

Reply:
Yep, can do!